ndcube/visualization/descriptor.py

```diff
@@ -44,5 +44,36 @@ def __get__(self, obj, objtype=None):
 
     def __set__(self, obj, value):
         if not isinstance(value, type):
-            raise TypeError("NDCube.plotter can only be set with an uninitialised plotter object.")
+            raise TypeError(
+                "Plotter attribute can only be set with an uninitialised plotter object.")
         setattr(obj, self._attribute_name, value(obj))
+
+
+class SequencePlotterDescriptor(PlotterDescriptor):
```

Comment:
I don't think you need this new class, just add the special case of `self._default_type == "mpl_sequence_plotter"` to the other one.